.modernizrrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "minify": true,
+  "options": [
+    "setClasses"
+  ],
+  "feature-detects": [
+    "test/webgl",
+    "test/css/flexbox"
```

Comment:
NVM, plz strike my comment, I ended up using flexbox to lay some stuff out cause absolute positioning for the general layout was causing my head to explode.